config/config.yaml

```diff
@@ -5,9 +5,10 @@ hash_length: 7
 
 # Specify the container framework used by each PRM wrapper. Valid options include:
 # - docker (default if not specified)
+# - podman - rootless version of docker, which should be a simple swap - if any algorithm doesn't work with podman, that is a bug.
```

Comment:
is it a bug with SPRAS or with the algorithm itself?

Reply:
Which error domain would you consider this to be a bug in? Does it signify a bug in SPRAS, or does it mean the algorithm/wrapper container itself is incompatible with Podman?

Reply:
It would mean the container itself is incompatible with Podman, but this likely means that on our side, we did some really hacky trick to get that container to work in the first place.